jenkins_pipelines/environments/common/pipeline-build-validation.groovy

```diff
@@ -5,7 +5,12 @@ def run(params) {
         env.resultdirbuild = "${resultdir}/${BUILD_NUMBER}"
         // The junit plugin doesn't affect full paths
         junit_resultdir = "results/${BUILD_NUMBER}/results_junit"
-        env.common_params = "--outputdir ${resultdir} --tf ${params.tf_file} --gitfolder ${resultdir}/sumaform"
+
+        if (!params.terraform_parallelism || params.terraform_parallelism == '') {
+            params.terraform_parallelism = 10
```

Comment:
Either we have a default value here, either we have a default value in Terracumber, both... I don't see it.

Reply:
Ok, then I would say we remove it here as use the default value from terracumber.